Flow.Launcher/ViewModel/ResultViewModel.cs

```diff
@@ -44,7 +44,7 @@ public ResultViewModel(Result result, Settings settings)
         private Settings Settings { get; }
 
         public Visibility ShowOpenResultHotkey =>
-            Settings.ShowOpenResultHotkey ? Visibility.Visible : Visibility.Hidden;
+            Settings.ShowOpenResultHotkey ? Visibility.Visible : Visibility.Collapsed;
```

Comment:
What's the different between these two?

Reply:
1. I increased the area of the hot key.
2. I tested the 'Do not display hotkey' option, but the hotkey area is reserved. ('Hidden' Property makes it like this.) In this case, the title area does not fit the width of the window. (it make a gap on the right side.)
3. the code make responsive width the title area and remove gap.

Reply:
Collapsed: Do not display the element, and do not reserve space for it in layout.
Hidden: Do not display the element, but reserve space for the element in layout.